### PR TITLE
isCrippled/isDmgLight/isDmgModerate/isDmgHeavy Debug Logging

### PR DESCRIPTION
**Pull request by Windchild292**
This adds some more debug logging for these methods in Aero and Tank, which I've found is required for testing.

### CODE REVIEW

File: megamek/src/megamek/common/GunEmplacement.java [hunk 1]
```diff
@@ -612,18 +594,18 @@ public void setDeployed(boolean deployed) {
...
         
         // we'll consider undeployed emplacements to be fully intact
-        if((occupiedStructure == null) || (initialBuildingCF + initialBuildingArmor == 0)) {
+        if ((occupiedStructure == null) || (initialBuildingCF + initialBuildingArmor == 0)) {
             return 1.0;
         }
         
-        return (occupiedStructure.getCurrentCF(getPosition()) + occupiedStructure.getArmor(getPosition())) / 
-                (initialBuildingCF + initialBuildingArmor);
+        return (1.0 * (occupiedStructure.getCurrentCF(getPosition()) + occupiedStructure.getArmor(getPosition())))
```
Comment (sixlettervariables) [inline megamek/src/megamek/common/GunEmplacement.java:608]: Is this a bug fix?

Comment (Windchild292) [reply megamek/src/megamek/common/GunEmplacement.java:608]: It is, it was trying to return a double from an int/int.

Comment (sixlettervariables) [reply megamek/src/megamek/common/GunEmplacement.java:608]: More obvious to cast the denominator to double. I presume this means that gun emplacements never showed the correct amount of armor remaining? (always 0)

Comment (Windchild292) [reply megamek/src/megamek/common/GunEmplacement.java]: Swapped, and I believe so. This was a catch from my IDE.

**Review by sixlettervariables — APPROVED** [review]
Looks good, sorry I missed this update!

**Comment by Windchild292** [comment]
No worries at all.

**Pull request merged**